app/javascript/packs/platform-authenticator-available.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  isWebauthnPlatformAuthenticatorAvailable,
+  isWebauthnPasskeySupported,
+} from '@18f/identity-webauthn';
+
+async function platformAuthenticatorAvailable() {
+  const platformauthenticatorAvailableInput = document.getElementById(
+    'platform_authenticator_available',
+  ) as HTMLInputElement;
+  if (!platformauthenticatorAvailableInput) {
+    return;
+  }
+  if (isWebauthnPasskeySupported() && (await isWebauthnPlatformAuthenticatorAvailable())) {
+    platformauthenticatorAvailableInput.value = 'true';
+  } else {
+    platformauthenticatorAvailableInput.value = 'false';
```

Comment:
Nit: Camel-case the two words "platform" "authenticator"

```suggestion
  const platformAuthenticatorAvailableInput = document.getElementById(
    'platform_authenticator_available',
  ) as HTMLInputElement;
  if (!platformAuthenticatorAvailableInput) {
    return;
  }
  if (isWebauthnPasskeySupported() && (await isWebauthnPlatformAuthenticatorAvailable())) {
    platformAuthenticatorAvailableInput.value = 'true';
  } else {
    platformAuthenticatorAvailableInput.value = 'false';
```